src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -833,8 +833,17 @@ public void CSharp_VerifyIDECodeStyleOptionsAreConfigurable()
 # IDE0046, PreferConditionalExpressionOverReturn
 dotnet_style_prefer_conditional_expression_over_return = true:silent
 
-# IDE0047
-No editorconfig based code style option
+# IDE0047, ArithmeticBinaryParentheses
+dotnet_style_parentheses_in_arithmetic_binary_operators = always_for_clarity:silent
```

Comment:
Isn't this the expected behavior even before this PR? Was this a bug somewhere that got fixed with this PR? or is there something I'm not seeing?

Reply:
Yes, the analyzer was not registering its supported options in its ctor, hence configuration code fixes were not working for it.